app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -1,13 +1,13 @@
 class StudentForm extends React.Component {
-  
+
```

Comment:
:P this is why all our devs should have 
```
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true,
	"ui_font_size_small": true,
	"ui_separator": true
}
``` in their sublime settings~~~!!!!!!!!!!!! @sonia-y